src/renderer/mesh_storage.zig

```diff
@@ -859,6 +853,61 @@ pub fn updateMeshes(targetTime: i64) void { // MARK: updateMeshes()
 	}
 }
 
+fn batchUpdateBlocks() void {
+	var lightRefreshList = main.List(*ChunkMesh).init(main.stackAllocator);
+	defer lightRefreshList.deinit();
+
+	var regenerateMesh = std.HashMapUnmanaged(chunk.ChunkPosition, *ChunkMesh, chunk.ChunkPosition.HashContext, 80){};
```

Comment:
Any reason why we can't just use an AutoHashMap?

Reply:
Also why not just make this a list anyways? Using a hashmap does seem premature since it only has a few entries.